### PR TITLE
first

### PR DESCRIPTION
**Pull request by anandayush81**

### CODE REVIEW

File: environment/SRRMSv2/api/status.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from flask import Blueprint
 
 statusBP = Blueprint('StatusAPI', __name__)
-
+#Added a comment -Ayush
```
Comment (Vishnu44d) [inline environment/SRRMSv2/api/status.py:4]: Redundent comment

**Review by Vishnu44d — COMMENTED** [review]
Please fix all comments

**Comment by anandayush81** [comment]
ok

**Comment by anandayush81** [comment]
fixed now

**Pull request merged**